html/pfappserver/lib/pfappserver/Form/Config/Syslog.pm

```diff
@@ -0,0 +1,92 @@
+package pfappserver::Form::Config::Syslog;
+
+=head1 NAME
+
+pfappserver::Form::Config::Syslog - Web form for an admin role
+
+=head1 DESCRIPTION
+
+Form definition to create or update an admin role
+
+=cut
+
+use strict;
+use warnings;
+use HTML::FormHandler::Moose;
+extends 'pfappserver::Base::Form';
+with 'pfappserver::Base::Form::Role::Help';
+use pf::constants::syslog;
+
+use pf::log;
+
+## Definition
+has_field 'id' =>
+  (
+   type => 'Text',
+   label => 'Syslog Name',
+   required => 1,
+   messages => { required => 'Please specify the name of the syslog entry' },
+  );
+has_field type => (
+    type => 'Hidden',
+    required => 1,
+    default_method => \&default_type,
+);
+has_field 'logs' =>
+  (
+   type => 'Select',
+   multiple => 1,
+   label => 'Logs',
+   options_method => \&options_logs,
+   default_method => \&default_logs,
+   element_class => [qw(chzn-select input-xxlarge)],
+   element_attr => {'data-placeholder' => 'Click to add a log'},
+   tags => { after_element => \&help,
+             help => 'Logs' },
+  );
+
+sub default_logs {
+    [ map { $_->{name} } @pf::constants::syslog::SyslogInfo ];
+}
+
+has_block  definition =>
```

Comment:
That block would be better placed above the default_logs subroutine so that its close to its fields friends

#:nail_care: